packages/react-native/src/Authenticator/common/FederatedProviderButton/FederatedProviderButton.tsx

```diff
@@ -13,18 +13,14 @@ export default function FederatedProviderButton({
   textStyle,
   ...rest
 }: FederatedProviderButtonProps): JSX.Element {
-  const containerStyle = useCallback(
-    ({ pressed }: PressableStateCallbackType): StyleProp<ViewStyle> => {
-      const pressedStateStyle =
-        typeof style === 'function' ? style({ pressed }) : style;
-      return [styles.container, pressedStateStyle];
-    },
-    [style]
-  );
+  const pressableStyle = usePressableOverrideStyle({
+    style,
+    themedStyle: styles.container,
+  });
 
   return (
-    <Button {...rest} style={containerStyle}>
-      {source ? <Icon source={source} style={styles.icon} size={20} /> : null}
```

Comment:
`source` is not optional so removed ternary op., it's lowering our unit test threshold.

Reply:
IMO `source` should be optional in case we were to allow custom social providers in the future. It just provides more flexibility